iosApp/Survey.xcodeproj/xcshareddata/xcschemes/Survey Staging.xcscheme

```diff
@@ -89,10 +89,10 @@
       </BuildableProductRunnable>
    </ProfileAction>
    <AnalyzeAction
-      buildConfiguration = "Debug Staging">
+      buildConfiguration = "DebugStaging">
    </AnalyzeAction>
    <ArchiveAction
-      buildConfiguration = "Release Staging"
+      buildConfiguration = "ReleaseStaging"
```

Comment:
Just curious, why do we have to introduce this change to support the new Moko library? Can you help elaborate a bit on the cause with some reference links maybe because I don't find any mention point about this in their [own repo](https://github.com/icerockdev/moko-resources)? I raise this because if this is a valid change, we might want to update our existing iOS template for supporting also 😁

Reply:
The script 
```
			shellScript = "\"$SRCROOT/../gradlew\" -p \"$SRCROOT/../\" :Shared:copyFrameworkResourcesToApp \\\n    -Pmoko.resources.PLATFORM_NAME=$PLATFORM_NAME \\\n    -Pmoko.resources.CONFIGURATION=$CONFIGURATION \\\n    -Pmoko.resources.BUILT_PRODUCTS_DIR=$BUILT_PRODUCTS_DIR \\\n    -Pmoko.resources.CONTENTS_FOLDER_PATH=$CONTENTS_FOLDER_PATH\n";
```
Will try to read the files for `CONFIGURATION` at `BUILT_PRODUCTS_DIR` which have some issue with `space` in the naming, something with the MOKO script is reading the `space` character as `break` and will not be able to find the autogenerated files.

Reply:
This looks like a scripting issue to me that the `space` needs to be converted to `"\ "` for it to work. What do you think about updating the `BUILT_PRODUCTS_DIR ` to support the new space representation by using the following [approach](https://stackoverflow.com/a/51406339)? 🤔

-> Instead of using the original `BUILT_PRODUCTS_DIR`, we can convert it to `NEW_BUILT_PRODUCTS_DIR`:
```
NEW_BUILT_PRODUCTS_DIR ="${BUILT_PRODUCTS_DIR// /\\ }"
```
=> Then we can update this to: `-Pmoko.resources.BUILT_PRODUCTS_DIR=$NEW_BUILT_PRODUCTS_DIR`. Let me know what you think 🙏